desktop/core/src/desktop/js/ko/components/ko.dropDown.js

```diff
@@ -33,7 +37,7 @@ const TEMPLATE = `
   </a>
   <!-- /ko -->
   <!-- ko if: !menuOnly && (dropDownVisible() && searchable) -->
-  <input class="hue-drop-down-input" type="text" data-bind="textInput: filter, attr: { 'placeHolder': inputPlaceHolder }, visible: dropDownVisible, style: { color: filterEdited() ? '#000' : '#AAA', 'min-height': '22px', 'margin-left': '10px' }"/>
+  <input class="hue-drop-down-input" data-testid = "database-drop-down-input" type="text" data-bind="textInput: filter, attr: { 'placeHolder': inputPlaceHolder }, visible: dropDownVisible, style: { color: filterEdited() ? '#000' : '#AAA', 'min-height': '22px', 'margin-left': '10px' }"/>
```

Comment:
Remove (or add another param like above).

Reply:
I have made the change on this line. Not entirely sure if this was what we wanted to do. Please have a look.